umbrella/README.md

```diff
@@ -0,0 +1,95 @@
+# `polkadot-sdk`
+
+`polkadot-sdk` is an umbrella crate re-exporting all other published crates, except for the example and fuzzing, plus some of the runtime crates like Rococo. Consider it the entry to the whole Polkadot and Substrate ecosystem.
```

Comment:
I think the word "umbrella" is not actually that well known in this context. We may need to explain it more first.

You can also omit Rococo, it will be phased out.